packages/itwin/property-grid/src/components/PropertyGridContent.tsx

```diff
@@ -197,3 +218,20 @@ function useFilterer({ showNullValues, filterText }: UseFiltererProps) {
 
   return compositeFilterer;
 }
+
+function useThrottled(func: () => void, timeout: number) {
+  const [isThrottled, setIsThrottled] = useState(false);
+  const isThrottledRef = useLatest(isThrottled);
+  const funcRef = useLatest(func);
+
+  return () => {
+    if (isThrottledRef.current) {
+      return;
+    }
+    setIsThrottled(true);
```

Comment:
Why is state needed? Isn't `isThrottledRef` enough for this?

Reply:
It is not, leftover after refactoring, removed.